### PR TITLE
[DOCFIX] Add more guidance on K8s port forwarding

### PR DESCRIPTION
**Pull request by jiacheliu3**

### CODE REVIEW

File: docs/en/deploy/Running-Alluxio-On-Kubernetes.md [hunk 1]
```diff
@@ -738,11 +738,24 @@ You can do more comprehensive verification following [Verify Alluxio]({{ '/en/de
...
+The pod `alluxio-master-$i` does not have to be on the node you are running this command.
+
+> Note: `i=0` for the the first master Pod. When running multiple masters, forward port for each
 master. Only the primary master serves the Web UI.
 
+For example, you are on a node with hostname `master-node-1` and you would like to serve
+the Alluxio master web UI on `master-node-1:8080`.
+Here's the command you can run:
+```console
+[alice@master-node-1 ~] $ kubectl port-forward --address 0.0.0.0 alluxio-master-$i 8080:19999
```
Comment (madanadit) [inline docs/en/deploy/Running-Alluxio-On-Kubernetes.md:754]: @jiacheliu3 i=1 for this example?

Comment (jiacheliu3) [reply docs/en/deploy/Running-Alluxio-On-Kubernetes.md]: Actually no, the pod can be running on another node, it doesn't matter.

**Review by madanadit — APPROVED** [review]

**Comment by madanadit** [comment]
Thanks @jiacheliu3.

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
alluxio-bot, cherry-pick this to branch-2.4 please

**Pull request merged**